src/xcode/ENA/ENA/Source/Scenes/ExposureSubmission/ExposureSubmissionNavigationController.swift

```diff
@@ -185,6 +185,7 @@ class ExposureSubmissionNavigationController: UINavigationController, UINavigati
 
 	override func viewWillDisappear(_ animated: Bool) {
 		super.viewWillAppear(animated)
+		self.homeViewController?.updateTestResultState()
```

Comment:
Can we do it in another way?

Reply:
is there a way that you would prefer? this was just the simplest way to ensure the home view is updated whenever we close the modally presented exposure submission navigator.

Reply:
you can review for example `stateHandler` in interactor and do it in similar way